static/css/main.ie.css

```diff
@@ -8111,8 +8114,7 @@ a.focus .summary_event-date {
 }
 
 .footer_nav-list .list_link {
-  margin-right: 1.36363636em;
-  font-size: 1.375em;
+  margin-right: 1.125em;
   font-family: "AvenirNextLTW01-Regular",Arial,sans-serif;
```

Comment:
Minor:
Not your edit, but if you're in there and want to add spaces after the commas, that'd be great.